### PR TITLE
added Blob support for export SVG (Client side only)

### PR DESCRIPTION
**Pull request by melalj**
No need to have a backend to export an SVG. It requires [FileSaver.js](https://github.com/eligrey/FileSaver.js) to make it work to make it work.

Related to #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,7 +308,7 @@ title: {
   - `continent`: Continent name that came from the GeoJSON map file.
 - `longitudeKey:` _(string, default: 'longitude')_ Column name of the longitude (from the dataset).
 - `latitudeKey:` _(string, default: 'latitude')_ Column name of the latitude (from the dataset).
-- `exportSvg:` _(string, default: null)_ URL endpoint to download the current visualization as SVG. Read more on the section export SVG. (more details on a the section "Export as SVG")
+- `exportSvg:` _(bool, default: false)_ Show button to download the current visualization as SVG. You'll need to import [FileSaver.js](https://github.com/eligrey/FileSaver.js) to make it work
```
Comment (Ths2-9Y-LqJt6) [inline README.md]: I didn't quite follow the "you'll need to import" instructions at first.  Maybe change to:

> - `exportSvg:` _(bool, default: false)_ Show button to download the current visualization as SVG. You'll need to  download [FileSaver.js](https://github.com/eligrey/FileSaver.js) and add a `<script src="filesaver.min.js">` to make it work.  You may also use the [CDN version](https://cdnjs.com/libraries/FileSaver.js) in your `<script>` source.

Comment (melalj) [inline README.md]: Fixed on 8cafa20

File: src/components/GeoMap.js [hunk 2]
```diff
@@ -530,16 +530,17 @@ export default class GeoMap {
   }
 
   exportSvg() {
+    if (!saveAs) {
+      throw new Error('MapTable: Missing FileSaver.js library');
```
Comment (Ths2-9Y-LqJt6) [inline src/components/GeoMap.js]: if you don't have `FileSaver.min.js` included I don't hit this line.  Instead in the console i see:

`Uncaught ReferenceError: saveAs is not definedexportSvg @ GeoMap.js:533`

**Comment by Ths2-9Y-LqJt6** [comment]
aside from the merge conflict to master, latest commit looks great!

**Pull request merged**